flink/src/main/scala/ru/itclover/tsp/mappers/PatternProcessor.scala

```diff
@@ -0,0 +1,117 @@
+package ru.itclover.tsp.mappers
+
+import java.lang
+
+import cats.Id
+import org.apache.flink.streaming.api.scala.function.ProcessWindowFunction
+import org.apache.flink.streaming.api.windowing.windows.Window
+import org.apache.flink.util.Collector
+import ru.itclover.tsp.io.TimeExtractor
+import ru.itclover.tsp.v2.Pattern.QI
+import ru.itclover.tsp.v2.{PState, Pattern, StateMachine, Succ}
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ListBuffer
+
+//case class PatternFlatMapper[E, State <: PState[Inner, State], Inner, Out](
```

Comment:
Dead comments, no need to store it (it's in history)

Reply:
Fixed